tests/checks/fusion-symlink.nf/.checks

```diff
@@ -49,12 +11,9 @@ fi
 #
 echo initial run
 $NXF_RUN -c .config
+RUN_ID=$($NXF_CMD log | tail -n1 | awk '{ print $7 }')
```

Comment:
Can we have concurrent `$NXF_RUN` executions in the same folder?

Reply:
No, the test runner does one test at a time

Reply:
if the goal is to isolate the tests, would not `uuidgen` be enough ?

Reply:
sure, even better. I will update it